CP-CPS.md

```diff
@@ -1215,8 +1215,7 @@ ISRG is not required to publicly disclose any audit finding that does not impact
 
 ISRG performs a quarterly internal audit of at least 3% of issuance since the last WebTrust audit period.
 The sample is randomly selected. Results are saved and provided to auditors upon request.
-
-In addition, ISRG conducts pre-issuance linting for all issuance.
+This audit includes linting of the selected certificates.
```

Comment:
Maybe I am misreading the diff, but it seems strange to have a single linebreak here. If it's the same paragraph as before, shouldn't it just be on the same line? If it's not, don't we need two line breaks?

Reply:
I was making it a single paragraph, and following the style of the two previous sentences in that paragraph, which each have their own line. But you're right that this doesn't match the style of the rest of the document, so I've brought them all onto a single line.